### PR TITLE
Make Travis generate OSX packages.

### PR DESCRIPTION
**Pull request by maximedenes**
The packages will be built only for main branches (not pull requests),
and are accessible via bintray: https://bintray.com/coq/coq

### CODE REVIEW

File: .bintray.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+    "package": {
+        "name": "coq",
+        "repo": "coq",
+        "subject": "coq"
+    },
+
+    "version": {
+        "name": "8.7~alpha"
```
Comment (Zimmi48) [inline .bintray.json]: Hum, this does not fully correspond to the proposal here: https://github.com/coq/coq/pull/483#issuecomment-306645169
@maximedenes Could you clarify the policy for naming versions? I don't understand why it was postponed for so long.

**Comment by maximedenes** [comment]
> Hum, this does not fully correspond to the proposal here:

What do you suggest in the context of bintray?



> @maximedenes Could you clarify the policy for naming versions? I don't understand why it was postponed for so long.

It is not the policy which was postponed, it is the implementation. The way version numbers are handled in our build infrastructure is very messy. I plan to push a final version of #483 today or Monday, it is almost ready.

**Comment by Zimmi48** [comment]
> I plan to push a final version of #483 today or Monday, it is almost ready.

OK very cool!

**Comment by maximedenes** [comment]
>> Hum, this does not fully correspond to the proposal here:

> What do you suggest in the context of bintray?

ping.

**Comment by Zimmi48** [comment]
> What do you suggest in the context of bintray?

Sorry I actually have no better suggestion. I just want to make sure that this is indeed the prefix that you plan to use in the version number (and not e.g. `8.6+alpha`) for consistency.

**Comment by maximedenes** [comment]
Is this about `~` vs `+`? I thought the previous discussion concluded that there was no difference.

**Comment by Zimmi48** [comment]
Indeed, but choose one and stick to it (I'm fine with any) ;-)

**Comment by maximedenes** [comment]
It turns out `8.7.0~alpha` is not a valid git tag, whereas `8.7.0+alpha` is, so switching to the later.

**Comment by Zimmi48** [comment]
Please how many times will I have to ask this? Don't use 8.7.0+alpha but 8.7+alpha. Same with beta. Some software will put 8.7.0+alpha after 8.7.0 while it will never be the case with 8.7+alpha (and I don't care about the separator, a separator can easily be changed to accommodate one's use).

**Comment by maximedenes** [comment]
Fixed.

**Comment by Zimmi48** [comment]
@maximedenes Isn't this PR required for preparing 8.7+beta?

**Pull request merged**